src/js/plugins/monitor.js

```diff
@@ -51,69 +51,69 @@ class Monitor extends Plugin {
         mainDiv.innerHTML = `<div class="title grid__col grid__col--8-of-8"></div>
         <table class="title grid__col grid__col--8-of-8">
         <tr><td><div class="text grid__col grid__col--6-of-8">Observables</div></td>
-        <td><div class="text grid__col grid__col--6-of-8">
-            <select id="observables" style="width: fit-content;"></select>                   </div></td><td rowspan="4">
+        <td><div class="text grid__col grid__col--6-of-8" style="width: 400px;">
+            <select id="observables"></select>                   </div></td><td rowspan="4">
         <div id="plugOutput"></div></td></tr><tr><td>
-        <div class="text grid__col grid__col--6-of-8">RestartThreshold</div></td><td><div class="text grid__col grid__col--6-of-8"><select id="restart" style="width: -webkit-fill-available;"> </select>
+        <div class="text grid__col grid__col--6-of-8">RestartThreshold</div></td><td><div class="text grid__col grid__col--6-of-8" style="width: 400px;"><select id="restart"> </select>
         </div> </td></tr><tr><td></td><td>
```

Comment:
I dont think we need <td>'s to begin with, you may want to remove these and just stick to the div's.